docs/virtual_machines/creating_vms.md

```diff
@@ -16,6 +16,80 @@ For a full set of flags and their description use the following command:
 virtctl create vm -h
 ```
 
+# Creating Instance Types
```

Comment:
Would you mind moving this to its own page or make it a part of the "Instance types and preferences" page? My concern is that this "Creating VirtualMachines" is one of the first introductory pages a new user would go through. Mixing an advanced topic such as instance types into it may make it overwhelming.

Reply:
You can leave here a note that "The flow can be simplified by using instance types" and reference the relevant page

Reply:
I strongly disagree with this, instance types aren't an advance feature and if anything we want to drive more initial users towards using them with `virtctl create vm` as anything more involved is likely better done in a full blown manifest.

Reply:
Apologies, I see what you mean in terms of ordering however, having this later after the initial `Creating VirtualMachines on a cluster`  section might make more sense.

Reply:
I see you point in advertising this as "the way to manage VMs". I just don't want to scare people away before they manage to create their first VM. +1 on moving the instance types section lower in the page, I would be happy to approve this then

Reply:
I have moved it below the `Creating VirtualMachines on a cluster`

Reply:
@opokornyy do you also want to update the link in https://github.com/kubevirt/user-guide/pull/705/files#diff-ba5436ef866e9da76ce83c188f741af99579d076a7f4a17247cbb448965cd032R176 to point directly to the new place?

Reply:
I would keep the link as it is, it should point to the usage of virtctl to create instance types, preferences, and virtual machines.